### PR TITLE
Investment performance on QQQ 

### PR DESCRIPTION
**Pull request by jajdfkkk**
test

### CODE REVIEW

**Comment by jajdfkkk** [comment]
Please check the ''Investment on QQQ.py'' file

File: week03/project-1-Tony_Hu/investment on QQQ.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
+import pandas as pd
+from matplotlib import pyplot as plt
+import warnings
+warnings.filterwarnings('ignore')
```
Comment (stargrep) [inline week03/project-1-Tony_Hu/investment on QQQ.py:4]: why do we need the warning?

File: week03/project-1-Tony_Hu/investment on QQQ.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+                else:
+                    temp = mydata['Shares'].loc[s] * mydata['OPEN'].loc[s]
+                    sold -= mydata['Shares'].loc[s]
+                    mydata['Shares'].loc[s] = 0
+                    # Update future asset cost because of selling
+                    mydata['Cost_Asset'].loc[t:] -= temp
+    # Update current asset and total cash received
+    mydata['Asset'].loc[t] = mydata['Shares_sum'].loc[t] * mydata['CLOSE'].loc[t]
+mydata['Cash'] = mydata['Cash_in'].cumsum()
+# Evaluating performance
```
Comment (stargrep) [inline week03/project-1-Tony_Hu/investment on QQQ.py:78]: can you add some comments about the comparison of performances?

File: week03/project-1-Tony_Hu/scheduled_investment.py [hunk 3]
```diff
@@ -0,0 +1,100 @@
+import pandas as pd
+import datetime as dt
+
```
Comment (stargrep) [inline week03/project-1-Tony_Hu/scheduled_investment.py:3]: are you using this file or the 'investment on QQQ.py' for logic execution?

File: .idea/misc.xml [hunk 4]
```diff
@@ -1,4 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (stargrep) [inline .idea/misc.xml:1]: to resolve merge conflict we can either remove these two .xml file or add them to .gitignore.

**Review by stargrep — CHANGES_REQUESTED** [review]

File: week03/project-1-Tony_Hu/investment on QQQ.py (hunk 1 above)
Comment (jajdfkkk) [reply week03/project-1-Tony_Hu/investment on QQQ.py:4]: To filter warnings in the board below

File: week03/project-1-Tony_Hu/investment on QQQ.py (hunk 2 above)
Comment (jajdfkkk) [reply week03/project-1-Tony_Hu/investment on QQQ.py]: Comparison has been added to the PR. I integrate them as one plot.

File: week03/project-1-Tony_Hu/scheduled_investment.py (hunk 3 above)
Comment (jajdfkkk) [reply week03/project-1-Tony_Hu/scheduled_investment.py:3]: This file is not used. I'm using the ‘investment on QQQ.py' file. When I submit it said I've made changes in this file, but I didn't make any change.

File: .idea/misc.xml (hunk 4 above)
Comment (jajdfkkk) [reply .idea/misc.xml]: I deleted 'misc.xml' and 'workspace.xml'

**Pull request closed without merging**